### DISCUSSION TITLE
Baking modules into STM32WL5

### DISCUSSION THREAD

**Discussion by davefes** [micropython/micropython#13155] [STM32 / Pyboard] [answered]
```
MicroPython v1.22.0-preview.208.gb5449b0f0.dirty on 2023-12-08; NUCLEO-WL55 with STM32WL55JCI7
Type "help()" for more information.
>>> 
>>> import lora
>>> dir(lora)
['__class__', '__name__', '__dict__', '__path__']
>>> dir(lora.WL55SubGhzModem)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'WL55SubGhzModem'
```
First thought was I would have to do this myself, but looking into the `build-NUCLEO_WL55/frozen/lora`  I see that they should be there.  What am I doing wrong?

**Comment by andrewleech** [comment, marked as answer]
Yep `manifest.py` is the standard way to manage this now, for docs see https://docs.micropython.org/en/latest/reference/manifest.html

If you copied the latest master copy of NUCLEO_WL55 board profile to base yours on it should have its own manifest with the lora libraries included, see where and how this was added in https://github.com/micropython/micropython/commit/af52e1ff24b0b543f6fdb3bc937713a6ab94d506

**Reply by davefes** [reply]
Thank you.
